app/models/rejection_reasons/details.rb

```diff
@@ -1,7 +1,24 @@
 class RejectionReasons
   class Details
     include ActiveModel::Model
+    WORD_COUNT = 100
 
     attr_accessor :id, :label, :text
+    validate :text_present, :word_count
+
+    def inflate(model)
+      @text = model.send(id)
+      self
+    end
+
+    def text_present
```

Comment:
We need to set a custom error attribute key (ie. `:safeguarding_details` instead of `:text`) 
I couldn't see an obvious way to achieve this with `validates` arguments alone as the attribute is inferred.